sample-app/src/components/VisualAutocomplete/VisualSearchBar.tsx

```diff
@@ -94,6 +113,35 @@ export default function VisualSearchBar({
     />
   }
 
+  function renderRecentSearches() {
+    const options = recentSearches?.getRecentSearches().map(result => {
```

Comment:
Same question about the necessity of the `?`

Reply:
similar as above comment, recentSearches may be undefined if hideRecentSearches is true

Reply:
if recentSearches is null should we just short circuit and return null? Otherwise we still render a DropdownSection just with an empty options array

Reply:
sure thing! updated